ext/systemd/puppet.service

```diff
@@ -6,6 +6,7 @@ After=basic.target network.target puppetmaster.service
 [Service]
 EnvironmentFile=-/etc/sysconfig/puppetagent
 EnvironmentFile=-/etc/sysconfig/puppet
+EnvironmentFile=-/etc/default/puppet
```

Comment:
I'd prefer to keep it like this. Although it would make sense if we were dealing with more paths that just `/etc/default` and `/etc/sysconfig`, I'd agree with you, but at the moment, the extra logic doesn't seem worth it.

That, and templating this with vanagon is painful (according to @haus)